perceval/backends/core/gerrit.py

```diff
@@ -390,6 +381,53 @@ def next_retrieve_group_item(self, last_item=None, entry=None):
 
         return next_item
 
+    def __execute(self, cmd):
+        """ Execute gerrit command"""
+
+        if self.from_archive:
+            response = self.__execute_from_archive(cmd)
+        else:
+            response = self.__execute_from_remote(cmd)
+
+        return response
+
+    def __execute_from_archive(self, cmd):
+
+        response = self.archive.retrieve(cmd, None, None)
+
+        if isinstance(response, RuntimeError):
+            raise response
+
+        return response
+
+    def __execute_from_remote(self, cmd):
+        """ Execute gerrit command with retry if it fails"""
+
+        data = None  # data result from the cmd execution
+
+        retries = 0
+
+        while retries < self.MAX_RETRIES:
+            try:
+                data = subprocess.check_output(cmd, shell=True)
+
+                break
+            except subprocess.CalledProcessError as ex:
+                logger.error("gerrit cmd %s failed: %s", cmd, ex)
+                time.sleep(self.RETRY_WAIT * retries)
+                retries += 1
+
+        if data is None:
```

Comment:
As this variable could be either a string or an exception, I think you should find another name for it. `result`, maybe?